### PR TITLE
Add capabilities

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:
This PR will add more doc about the controller capabilities ,user should know the 
capability of the csi controller ,e.g what's it can do 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
N/A
**Special notes for your reviewer**:
N/A
**Release note**:
no release notes needed because it's doc change

### CODE REVIEW

**Comment by dims** [comment]
/ok-to-test

cc @adisky

File: docs/using-cinder-csi-plugin.md [hunk 1]
```diff
@@ -59,6 +59,14 @@ $ csc identity plugin-capabilities --endpoint tcp://127.0.0.1:10000
 CONTROLLER_SERVICE
 ```
 
+#### Get controller implemented capabilities
+```
+$ csc controller get-capabilities  --endpoint tcp://127.0.0.1:10000
```
Comment (adisky) [inline docs/using-cinder-csi-plugin.md]: Thanks for improving CSI docs, Are you able to get the output?? as i cannot see ControllerGetCapabilities implemented here 
https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/csi/cinder/controllerserver.go

Comment (jichenjc) [reply docs/using-cinder-csi-plugin.md]: actually, the output is listed here: 

&{type:CREATE_DELETE_VOLUME }
&{type:PUBLISH_UNPUBLISH_VOLUME }

so I believe it's the parent class handled this request and seems no need for us to override this method?

Comment (jichenjc) [reply docs/using-cinder-csi-plugin.md]: I mean , the command does give the output as listed above....

Comment (FengyunPan2) [reply docs/using-cinder-csi-plugin.md]: IMO, 'output a readable example' should be more helpful, like Line 75.

Comment (adisky) [reply docs/using-cinder-csi-plugin.md]: @jichenjc ok, so it think we are getting the default output
@FengyunPan2 IMHO, we should show the actual ouput in doc also, and the actual output has this weird format, so i think it is fine as of now.

Comment (FengyunPan2) [reply docs/using-cinder-csi-plugin.md]: +1 : show the actual output

**Comment by dims** [comment]
/override openlab/cloud-provider-openstack-acceptance-test-flexvolume-cinder

**Pull request merged**